engines/scumm/scumm.cpp

```diff
@@ -3008,6 +3057,454 @@ void ScummEngine::versionDialog() {
 	runDialog(*_versionDialog);
 }
 
+#ifdef ENABLE_SCUMM_7_8
```

Comment:
Please move it to a separate file, maybe `gfx_gui.cpp` Leave it in the same class, just `scumm.cpp` has enough stuff already. (Also, if there are other parts which could be moved there, please do that as well).

Reply:
You are right, I will do that. After all, if (eventually) all the original GUIs are going to be available, it wouldn't be appropriate to clutter `scumm.cpp` with the relevant logic.